src/style/markdown.css

```diff
@@ -395,14 +395,16 @@
 		}
 
 		content-region[name*='guide'] & {
+			display: flex;
+			align-items: center;
+
 			a {
 				opacity: 0;
-				display: inline-flex;
 				width: 1.5rem;
 				margin-left: -1.5rem;
 
-				&::before {
-					content: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="%23fff" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><path d="M15 7h3a5 5 0 0 1 5 5 5 5 0 0 1-5 5h-3m-6 0H6a5 5 0 0 1-5-5 5 5 0 0 1 5-5h3"></path><line x1="8" y1="12" x2="16" y2="12"></line></svg>');
+				svg {
+					display: block;
```

Comment:
Not really accessibility related, but I noticed the `::before` doesn't vertically align correctly and I couldn't for the life of me fix it. No combination seemed to work.

Refactored into a `<use>` instead.